live-examples/js-examples/date/date-gettimezoneoffset.js

```diff
@@ -3,7 +3,7 @@ const date2 = new Date('August 19, 1975 23:15:30 GMT-02:00');
 
 console.log(date1.getTimezoneOffset());
 // expected output: your local timezone offset in minutes
-// (eg -120). NOT the timezone offset of the date object.
+// (Eg -120). NOT the timezone offset of the date object.
```

Comment:
```suggestion
// (e.g., -120). NOT the timezone offset of the date object.
```
- the "e.g.," contraction should begin in lower case
- Should "NOT" be "Note"?

Reply:
I have corrected `e.g.,`. Word `NOT` is fine in this context, because `getTimezoneOffset` always returns the timezone offset of a user(considering his localization). Values inside the Date object might matter, because of Dayling Saving Time, but it's explained by an article in detail, so I think it's fine to keep it simple in here.